src/qt/editor/inviwomainwindow.cpp

```diff
@@ -957,6 +1014,38 @@ void InviwoMainWindow::saveWorkspaceAsCopy() {
     saveWindowState();
 }
 
+void InviwoMainWindow::showWelcomeScreen() {
+    if (!welcomeWidget_) {
+        welcomeWidget_ = new WelcomeWidget(this, centralWidget_);
+
+        centralWidget_->setUpdatesEnabled(false);
+        centralWidget_->setCurrentIndex(0);
+        centralWidget_->insertTab(0, welcomeWidget_, "Get Started");
+        centralWidget_->setUpdatesEnabled(true);
+
+        QToolButton* closeBtn = new QToolButton();
+        closeBtn->setIcon(QIcon(":/stylesheets/images/close.png"));
+        closeBtn->setObjectName("dockBtn");
+        QObject::connect(closeBtn, &QToolButton::clicked, this, [&]() { hideWelcomeScreen(); });
+
+        centralWidget_->tabBar()->setTabButton(0, QTabBar::RightSide, closeBtn);
+    }
+    centralWidget_->setCurrentIndex(centralWidget_->indexOf(welcomeWidget_));
+    welcomeWidget_->setFocus();
+}
+
+void InviwoMainWindow::hideWelcomeScreen() {
+    if (welcomeWidget_) {
+        int index = centralWidget_->indexOf(welcomeWidget_);
+        if (index > -1) {
+            centralWidget_->removeTab(index);
+        }
+
+        welcomeWidget_->close();
```

Comment:
Should we close it now or just hide?

Reply:
either is fine, I guess.

Reply:
or close will not delete it? what does close even mean?

Reply:
close implicitly deleted the widget, will replace it with `hide()` and keep the widget.